glean-core/rlb/src/system.rs

```diff
@@ -73,6 +73,22 @@ pub fn get_os_version() -> String {
     parse_linux_os_string(whatsys::kernel_version().unwrap_or_else(|| "Unknown".to_owned()))
 }
 
+#[cfg(target_os = "windows")]
+/// Returns the Build ID for Windows kernels versions >= 10.0
```

Comment:
```suggestion
/// Returns the Windows Build Number.
```

The build number exists on Windows < 10 as well.